src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ client:
   frontedservers: []
   chainedservers: {{range .fallbacks}}
     fallback-{{.ip}}:
-      addr: {{.ip}}:443
+      addr: {{.ip}}
```

Comment:
Ok http://www.yaml.org/spec/1.2/spec.html#id2759963 says that `Mappings use a colon and space (“: ”) to mark each key: value pair`, so it's legal yaml.